frontend/e2e-test/test/e2e/dev-api/types.ts

```diff
@@ -471,6 +475,44 @@ export interface Decision {
   endDate: ISODate
 }
 
+export interface Decision {
+  id: UUID
+  type: DecisionType
+  startDate: LocalDate
+  endDate: LocalDate
+  unit: DecisionUnit
+  applicationId: UUID
+  childId: UUID
+  childName: string
+  documentUri: string
+  decisionNumber: number
+  sentDate: LocalDate
+  status: DecisionStatus
+}
+
+export interface DecisionUnit {
+  id: UUID
+  name: string
+  daycareDecisionName: string
+  preschoolDecisionName: string
+  manager: string | null
+  streetAddress: string
+  postalCode: string
+  postOffice: string
+  approverName: string
+  decisionHandler: string
+  decisionHandlerAddress: string
+}
+
+export function deserializeDecision(json: JsonOf<Decision>): Decision {
+  return {
+    ...json,
+    startDate: LocalDate.parseIso(json.startDate),
+    endDate: LocalDate.parseIso(json.endDate),
+    sentDate: LocalDate.parseIso(json.sentDate)
+  }
+}
```

Comment:
I think these are now duplicated in 3 places (citizen-frontend, employee-frontend and e2e-tests), so if there's enough other cases like this, at some point we may want to think about sharing api-typings somehow

Reply:
Yes!